<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldDesccriptor</class>
 <widget class="QWidget" name="FieldDesccriptor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>135</width>
    <height>64</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="sbValue"/>
      </item>
      <item>
       <widget class="QSpinBox" name="sbValue2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="leResult">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="leFieldName"/>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="cbType"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
